packages/pkg/src/index.ts

```diff
@@ -27,7 +27,7 @@ const {
 
 const format = (properties: ObjectProperty[], options: FormatOptions) => {
   let props = ['engines', 'devEngines', 'scripts', ...dependencyNames].reduce(
```

Comment:
I'd expect to change this line instead. 🤔

Reply:
done